src/Microsoft.Azure.SignalR.Common/ServiceConnections/ClientConnectionScopeInternal.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Dynamic;
+using System.Threading;
+
+namespace Microsoft.Azure.SignalR
+{
+
+    internal class ScopePropertiesAccessor<TProps>
```

Comment:
convention: per class per file, and don't have nested classes except when the nested is private

Reply:
Ok!